third-party/realdds/include/realdds/topics/ros2/ros2time.h

```diff
@@ -12,7 +12,7 @@
 #define _FAST_DDS_GENERATED_STD_MSGS_MSG_TIME_H_
 
 
-#include <fastrtps/utils/fixed_size_string.hpp>
+//#include <fastrtps/utils/fixed_size_string.hpp>
```

Comment:
Was this line something we added or was it downloaded from the net like this?
This is an auto generated file so not recomended to change manually

Reply:
I've done it elsewhere and this file is not expected to change.
If it does, I usually do a diff and make sure everything aligns.
Up to you, but I prefer it without the include

Reply:
Please restore. I don't want to touch third-party files.
Also we don't clean all unneeded includes, just this one.

Reply:
As discussed offline:
We change the headers anyway: we change the license and we have to change the header names and locations.
I just removed the eProsima headers that are not needed; the standard headers, even if not in use, should be OK.